documentation/manual/source/_theme.indigo/layout.html

```diff
@@ -42,13 +45,17 @@
           <a class="active" href="/current">docs</a>
         </li>
         <li>
+         <a class="" href="/current/apiviewer">api</a>
+        </li>
+        <li>
           <a class="" href="http://stackoverflow.com/questions/tagged/qooxdoo">support</a>
         </li>
         <li>
           <a class="" href="https://gitter.im/qooxdoo/qooxdoo">gitter</a>
```

Comment:
It's not one of the PR's changed lines, but...

I suspect that few know what gitter is. How about changing the visible text to "chat room" or "interactive discussion" or some such thing?